pokedexForm.php

```diff
@@ -2,9 +2,9 @@
     <body>
```

Comment:
this file is nothing to do with story 1, please delete

Reply:
Have removed said file plus some others which weren't needed